src/main/java/calculator/calculate/ParsingString.java

```diff
@@ -0,0 +1,81 @@
+package calculator.calculate;
+import calculator.global.GlobalConstant;
+import calculator.exception.Exception;
+import java.util.ArrayList;
+
+public class ParsingString {
+    private ArrayList<Character> delemeter;
+    private String check_slash = "";
+    private String check_line = "";
+    private char adding_delemeter;
+    private ArrayList<Integer> numlist = new ArrayList<Integer>();
+    private String input;
+    private String parsingInput;
+
+    private void AddDelemeter() {
+        if (LenCheck()) return;
+        ParsingPattern();
+        if (check_slash.equals("//") && check_line.equals("\\n")) {
+            Exception.IsDelemeterChar(input.charAt(2));
+            adding_delemeter = input.charAt(2);
+            delemeter.add(adding_delemeter);
+            GlobalConstant.StartIndex = 5;
+        }
+    }
+
+    private void ParsingPattern(){
+        check_slash = input.substring(0,2);
+        check_line = input.substring(3,5);
+    }
+
+    private boolean LenCheck(){
+        if(input.length() < 5) return true;
+        else return false;
+    }
```

Comment:
[Google Java Style Guide](https://google.github.io/styleguide/javaguide.html#s4.1.1-braces-always-used)에 맞춰보시면 어떨까 싶어요!
> Braces are used with `if`, `else`, `for`, `do` and `while` statements, **even when the body is empty or contains only a single statement.**

Reply:
추가로 자바 메서드명은 동사로 시작하는 것을 권장하고 있어요.
또한 `length`를 `len`으로 줄여주셨는데, 축약어는 최대한 지양하는 것이 코드를 읽기 더 편해진다고 생각합니다!

```suggestion
    private boolean checkLength(){
        if(input.length() < 5) return true;
        else return false;
    }
```